code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -100,7 +100,7 @@
 	if(I.force)
 		var/attack_direction = get_dir(user, src)
 		apply_damage(I.force, I.damtype, affecting, wound_bonus = I.wound_bonus, bare_wound_bonus = I.bare_wound_bonus, sharpness = I.get_sharpness(), attack_direction = attack_direction, attacking_item = I)
-		if(I.damtype == BRUTE && IS_ORGANIC_LIMB(affecting))
+		if(I.damtype == BRUTE && affecting.can_bleed())
```

Comment:
Unrelated, but given how granular we're getting with individual limbs bleeding, maybe we should eventually add a chance for a small/soft metal clang if you attack a metallic limb.